src/modules/poweraccent/PowerAccentKeyboardService/KeyboardListener.cpp

```diff
@@ -112,6 +118,11 @@ namespace winrt::PowerToys::PowerAccentKeyboardService::implementation
         }
     }
 
+    bool KeyboardListener::IsSuppressedByGameMode()
+    {
+        return m_settings.doNotActivateOnGameMode && detect_game_mode();
```

Comment:
I like how we check the Setting first so we don't have to run detect_game_mode to save on cycles in the LLKH.